File: Python Code/Daily_Flash_Code_0/Week13/17Apr/76-DailyFlash_Solutions/17_Apr_Solution_Three/Python/prog1.py
str1 = input()

str2 = input()

m = min(len(str1), len(str2))
flag = 0
for i in range(m):
    if(str1[i] != str2[i]):
        print(ord(str1[i]) - ord(str2[i]))
        flag = 1
        break
if(flag == 0):
    print("Equal")

File: Python Code/Daily_Flash_Code_0/Week13/17Apr/76-DailyFlash_Solutions/17_Apr_Solution_Three/Java/prog1.java
import java.util.*;

class Demo {
	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		String str1 = sc.next();
		String str2 = sc.next();
		int min;
		if(str1.length() > str2.length())
			min = str2.length();
		else
			min = str1.length();
		for(int i = 0; i < min; i++) {
			if(str1.charAt(i) != str2.charAt(i)) {
				System.out.println((int)(str1.charAt(i) - str2.charAt(i)));
				return;
			}
		}
		System.out.println("Equal.");

	}
}
